packages/web3-eth-accounts/src/wallet.ts

```diff
@@ -109,6 +223,42 @@ export class Wallet<
 		return false;
 	}
 
+	/**
+	 * Securely empties the wallet and removes all its accounts.
```

Comment:
```suggestion
	 * Securely empties the wallet and removes all its accounts. Use this with *caution as it will remove all accounts stored in local wallet. 
```